<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnui">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>NoIUI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCon">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>QbjCon</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLambda">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>btnLambda</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFortch">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>btnFortch</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
